content/learn/book/getting-started/plugins/_index.md

```diff
@@ -35,6 +35,37 @@ You should hopefully notice two things:
 * **A window should pop up**. This is because we now have {{rust_type(type="struct" crate="bevy_window" name="WindowPlugin")}}, which defines the window interface (but doesn't actually know how to make windows), and {{rust_type(type="struct" crate="bevy_winit" name="WinitPlugin")}} which uses the [winit library](https://github.com/rust-windowing/winit) to create a window using your OS's native window API.
 * **Your console is now full of "hello" messages**: This is because {{rust_type(type="struct" crate="bevy" name="DefaultPlugins")}} adds an "event loop" to our application. Our App's ECS Schedule now runs in a loop once per "frame". We will resolve the console spam in a moment.
 
+## Rendering a window
+The output of everything done to this point was only on console. But ofcourse bevy isn't restricted to that so, let's render a window. Change your importations to this:
+
+```rs
+use bevy::{prelude::*, window::*};
+```
+
+Then edit your `main` function with the following:
+
+```rs
+fn main() {
```

Comment:
I think this type of "window configuration" belongs in a dedicated window section. This brings in too many windowing details too early in the learning path. This page should just have the minimal details required to define a plugin.

Reply:
Probably best to hold off until we start working on the next book version. @alice-i-cecile any thoughts here?